Blockzilla/OverlayView.swift

```diff
@@ -257,6 +278,28 @@ class OverlayView: UIView {
             self.isUserInteractionEnabled = true
         }
     }
+    
+    func setSearchSuggestionsPromptViewDelegate(delegate: SearchSuggestionsPromptViewDelegate) {
+        searchSuggestionsPrompt.delegate = delegate
+    }
+    
+    func hideSearchSuggestionsPrompt() {
```

Comment:
Can we collapse these functions into something like ```displaySearchSuggestionsPrompt(hide: Bool)``` where we handle the logic for both inside based on the value of ```hide```? I think this would lead to less code duplication.